yaml2ncml/yaml2ncml.py

```diff
@@ -102,42 +108,41 @@ def add_var_atts(text, a):
     for var in vars:
         text += '<variable name="{:s}">\n'.format(var)
         try:
-            text += str_att('standard_name',cf[var])
+            text += attribute_name('standard_name', cf[var])
         except:
             pass
-        text += str_att('grid','grid')
-        text += str_att('content_coverage_type','modelResult')
+        text += attribute_name('grid', 'grid')
+        text += attribute_name('content_coverage_type', 'modelResult')
         if var in rho_vars:
-            text += str_att('location','face')
+            text += attribute_name('location', 'face')
         elif var in u_vars:
-            text += str_att('location','edge1')
+            text += attribute_name('location', 'edge1')
         elif var in v_vars:
-            text += str_att('location','edge2')
+            text += attribute_name('location', 'edge2')
         text += '</variable>\n\n'
     return text
 
-def write_grid_var(text):
-    grid_var="""<variable name="grid" type="int">
-        <attribute name="cf_role" value="grid_topology"/>
-        <attribute name="topology_dimension" type="int" value="2"/>
-        <attribute name="node_dimensions" value="xi_psi eta_psi"/>
-        <attribute name="face_dimensions"
-            value="xi_rho: xi_psi (padding: both) eta_rho: eta_psi (padding: both)"/>
-        <attribute name="edge1_dimensions" value="xi_u: xi_psi eta_u: eta_psi (padding: both)"/>
-        <attribute name="edge2_dimensions" value="xi_v: xi_psi (padding: both) eta_v: eta_psi"/>
-        <attribute name="node_coordinates" value="lon_psi lat_psi"/>
-        <attribute name="face_coordinates" value="lon_rho lat_rho"/>
-        <attribute name="edge1_coordinates" value="lon_u lat_u"/>
-        <attribute name="edge2_coordinates" value="lon_v lat_v"/>
-        <attribute name="vertical_dimensions" value="s_rho: s_w (padding: none)"/>
-    </variable>\n """
+
+def write_grid_var(text, template='roms'):
+    if template == 'roms':
+        tpl = 'roms.tpl'
+    else:
+        raise ValueError('Template {!r} is not implemented!'.format(template))
+
```

Comment:
@rsignell-usgs This is how it would look like to open from a template.  My guess is that more then just the `grid_var` could be in the template.